content/en/real_user_monitoring/guide/understanding-the-rum-event-hierarchy.md

```diff
@@ -0,0 +1,114 @@
+---
+
+## Overview
+
+This guide walks through the different [types of data][1] that RUM collects and describes the hierarchy of each event type. See the diagram below for a high level overview of the hierarchy.
+
+{{< img src="real_user_monitoring/guide/understanding-rum-event-hierarchy/rum-session-hierarchy-overview.png" alt="Diagram of the RUM event hierarchy." style="width:50%;">}}
+
+## Sessions
+All RUM data refers to user or synthetics sessions, which are at the top of the event hierarchy. A session is a unique user journey and encompasses everything (e.g., pages viewed, views, clicks, scrolls, errors) the user triggered. A session can last up to 4 hours of continuous activity, or it can expire after [15 minutes of inactivity][2]. Since a session encompasses the entire journey, all [attributes][3] tied to that user are also tied to that session. For example, you may want to query on a default attribute, like `application.id`, then add something more custom, like [user attributes][4].
+
+#### Sample search: List all sessions from user "Jane Doe"
+
+To list all sessions from a specific user, select **Sessions** from the event type dropdown, then make a search query for the user and application.
+
+{{< img src="real_user_monitoring/guide/understanding-rum-event-hierarchy/rum-sample-search-list-all-sessions.png" alt="Sample search listing all sessions from user "Jane Doe"." style="width:80%;">}}
+
+Notice that each session is automatically associated with a unique `session.id`.
+
+## Views
+Within a session, a view event is created each time a user navigates to a page (Browser RUM) or screen/screen segment (Mobile RUM) of an application. 
+
+Each view automatically collects multiple view specific attributes and data, such as text in the URL and timing metrics, like the load time of a given page. When querying for specific views, you can add any default level attributes, like device, operating system, user information, etc. However, for event-specific attributes, these must be view-specific. To switch over to view events only, you can adjust the event selector as shown in the video below.
+
+{{< img src="real_user_monitoring/guide/understanding-rum-event-hierarchy/rum-switch-views.mp4" alt="RUM views" video="true" style="width:80%;">}}
+
+Similarly to the `session.id`, each view automatically has a unique `view.id` connected to it. 
+
+### Actions, errors, resources, and long tasks
+
+Within views, the SDK creates more granular events that all fall along the same level hierarchically. However, each event is unique and carries its own attributes and properties.
+
+### Actions
+
+Actions represent user activity on a page. In browsers, all click actions are automatically collected. On mobile, all taps, swipes, and scrolls are collected. Beyond these default actions, you can also send [custom actions][5], such as form completion and business transactions. 
+
+#### Sample search: Top list of all "Add to cart" actions that led to an error
```

Comment:
This header feels kind of redundant given the text is almost identical on the next line
```suggestion
#### Sample search: Top list of actions leading to an error
```